### ISSUE TITLE
help!?

### ISSUE THREAD

**Issue by mrwulff** [mbrockman1/Kivy_Notification#1]
Just found this project.  How do i build it so that it overwrites just the notification parts of plyer

**Comment by mbrockman1** [comment]
The way i would do it is this:
1) Add the original plyer folder from kivy/plyer project to the root directory of you project.
2) add my notifications.py from plyer/platforms/ios to the kivy/plyer project
3) Add my macoslib and ioslib folders to the kivy/plyer directory in your project.
4) add the shell script i wrote into your plyer directory of your project as well
5) run the shell script I wrote once everything is set. This will may a dylib, which using buildozer, I had to add the .dylib filetype to be included.
6) debug it and see if it works. Let me know, I will support you as best as I can. But the issue is since plyer for ios has been kind of abandoned try using this: https://github.com/psychowasp/KivySwiftLink It is way more robust and future proof than plyer/objC

**Comment by mrwulff** [comment]
thanks for the reply.  So I was definently missing the script to build the dylib.  However now when i do build it i get an error right away in xcode saying it was built for the sim and ios.

***.xcodeproj Building for iOS, but the linked and embedded library 'UniBlocks.dylib' was built for iOS + iOS Simulator.


Is there a way to only build it for ios?

**Comment by mbrockman1** [comment]
https://stackoverflow.com/questions/65303304/xcode-12-3-building-for-ios-simulator-but-the-linked-and-embedded-framework-wa

**Comment by mbrockman1** [comment]
You are taxing my brain hahaha. I havent worked on this in some time. Try the Build Settings ->"Validate Workspace" step.

**Comment by mrwulff** [comment]
Yeah for now im just using the ios file, but am having a hard time 'finding' it.  How exactly do i put it in my build.  Do i just stick it in my "YourApp" folder, or do i add it to xcode frameworks.  Also I'm not using buildozer.  Should I in general?

**Comment by mbrockman1** [comment]
Glad you got something to work. But yes, I know I keep saying it but check out https://github.com/psychowasp/KivySwiftLink. The reasoning behind it is, from my usage of Pyobjus, you effectively have to learn/write obj-C code in attempts to try to get anything to work. Apple redid the notification center which is a nightmare to use python->pyobjus->objc. I still haven't had any success. The KivySwiftLink allows for Python->Swift which Swift is MUCH easier to write than objc and the documentation is way better. The catch with KivySwiftLink is it is being developed by 1 guy and hasn't been adopted by the kivy team so development is understandably slow.
